concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/20 Sort Data/6 Sort Field Values by Summary Values.md

```diff
@@ -0,0 +1,136 @@
+Users can sort field values by totals and grand totals of the opposite dimension. For instance, to sort column field values, users should right-click a row field value and select one of the commands from the context menu:
+
+![DevExtreme PivotGrid: Sort field values by summary values](/images/pivotgrid/sorting-by-summary-values.png)
+
+Set the [allowSortingBySummary](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#allowSortingBySummary) property to **true** to enable this functionality for all fields. You can also set the same property for an individual field if you want to override the common setting.
+
+To sort field values by grand totals programmatically, set the [sortBySummaryField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#sortBySummaryField) property to the data field to sort by. If you want to sort field values by totals, you should also specify the [sortBySummaryPath](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#sortBySummaryPath) property. To control the sort order, use the [sortOrder](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#sortOrder) property.
```

Comment:
To sort field values by grand totals programmatically, set the [sortBySummaryField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#sortBySummaryField) property to the data field to sort by.

This sentence doesn't really make sense and is poorly said. For example, you mention "to sort" twice. Can you rewrite it?